choice_reaction_time/experiment.js

```diff
@@ -283,8 +283,8 @@ var start_practice_block = {
 var start_test_block = {
   type: 'poldrack-text',
   timing_response: 60000,
-  data: {exp_id: 'choice_reaction_time', trial_id: 'test_intro'},
-  text: '<div class = centerbox><p class = block-text>We will now begin the first test block. You will no longer receive feedback about your responses.</p><p class = block-text>If you see the <font color="orange">orange</font> square you should press the <strong>' + correct_responses[0][0] + '</strong> key. If you see the <font color="blue">blue</font> square you should press the <strong>' + correct_responses[1][0] + '</strong> key. Press <strong>enter</strong> to begin.</p></div>',
+  data: {exp_id: 'choice_reaction_time', trial_id: 'practice_intro'},
```

Comment:
Add back in your correction (you correctly changed "practice_intro" to "test_intro". Also remove "start_test_block2" since it is no longer used.